documentation/API_documentation/device-swap-API-Readiness-Checklist.md

```diff
@@ -0,0 +1,29 @@
+# API Readiness Checklist
+
+Checklist for device-swap 0.2.0-rc.1 in r2.1
+
+| Nr | API release assets  | alpha | release-candidate |  initial<br>public | stable<br> public | Status | Comments |
+|----|----------------------------------------------|:-----:|:-----------------:|:-------:|:------:|:----:|:----:|
+|  1 | API definition                               |   M   |         M         |    M    |    M   |   Y  | [link](code/API_definitions/device-swap.yaml) |
+|  2 | Design guidelines from Commonalities applied |   O   |         M         |    M    |    M   |   Y  |      |
+|  3 | Guidelines from ICM applied                  |   O   |         M         |    M    |    M   |   Y  |      |
+|  4 | API versioning convention applied            |   M   |         M         |    M    |    M   |   Y  |      |
+|  5 | API documentation                            |   M   |         M         |    M    |    M   |   Y  | Embed documentation into API spec - [link](/code/API_definitions/device-swap.yaml) |
+|  6 | User stories                                 |   O   |         O         |    O    |    M   |   Y  | [check](/documentation/API_documentation/device-swap_check_User_Story.md) [retrieve](/documentation/API_documentation/device-swap_retrieve_User_Story.md) |
+|  7 | Basic API test cases & documentation         |   O   |         M         |    M    |    M   |   Y  | [check](/code/Test_definitions/device-swap-check.feature) [retrieve](/code/Test_definitions/device-swap-retrieve-date.feature) |
+|  8 | Enhanced API test cases & documentation      |   O   |         O         |    O    |    M   |   N  | pending |
+|  9 | Test result statement                        |   O   |         O         |    O    |    M   |   N  |  Spring25 EXCEPTION: Test results not available (*)    |
```

Comment:
That exception is for Spring25 not yet decided within the TSC ... and the test result statement is only needed for public releases stable API, not for the release candidates and also not for  initial API versions like 0.2.0.
Therefore:
```suggestion
|  9 | Test result statement                        |   O   |         O         |    O    |    M   |   N  |           |
```